app/views/help/missing_cases.html.erb

```diff
@@ -0,0 +1,51 @@
+<% content_for :title, 'Missing cases – Digital Prison Services' %>
+
+<%= render 'breadcrumbs', trail: { '/' => 'Home', help_dashboard_path => 'Help' }, current: 'Missing cases' %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l govuk-!-margin-bottom-6">
+      Missing cases
+    </h1>
+
+    <h2 class="govuk-heading-m govuk-!-margin-bottom-2">
+      Information missing on NOMIS or NDelius
+    </h2>
+    <p>
+      Make sure information about the case has been entered correctly on NOMIS.
+      For example, this might include sentence calculation dates.
+    </p>
+    <p>
+      Also, if you can use the service to allocate, check if the case is listed on
+      <a class="govuk-link govuk-link--no-visited-state" href="<%= missing_information_prison_prisoners_path(@prison_code) %>">Add missing details</a>.
```

Comment:
Could this be a _link_to_?

Reply:
The convention in the app is to do it this way, so I'm sticking with that. I know, it's weird, but I'd rather it all looked the same.